### PR TITLE
Improve UPS client with methods to get and delete a single attribute

### PR DESCRIPTION
**Pull request by jcazevedo**
This PR improves the UPS client with methods to get and delete a single attribute of an user.

The get single attribute method avoids fetching the whole attribute set if the client is only interested in one attribute. The delete single attribute method avoids having to fetch the whole attribute set first, filter the attribute one wants to delete, and update the whole attribute set afterwards.

### CODE REVIEW

File: src/main/scala/eu/shiftforward/adstax/ups/api/UserProfileStorageResponse.scala [hunk 1]
```diff
@@ -19,6 +19,18 @@ object GetUserResponse {
 }
 
 /**
+ * The response for a [[GetUserAttribute]] request.
+ *
+ * @param attribute the attribute value for the requested key wrapped in a
```
Comment (ruippeixotog) [inline src/main/scala/eu/shiftforward/adstax/ups/api/UserProfileStorageResponse.scala]: This should be named `attributeValue`.

File: src/main/scala/eu/shiftforward/adstax/ups/api/UserProfileStorageResponse.scala [hunk 2]
```diff
@@ -19,6 +19,18 @@ object GetUserResponse {
...
+ * `Some` if the user and the attribute exists, `None` otherwise
```
Comment (ruippeixotog) [inline src/main/scala/eu/shiftforward/adstax/ups/api/UserProfileStorageResponse.scala]: Minor nitpick: can you wrap this in 120 columns?

File: build.sbt [hunk 3]
```diff
@@ -3,7 +3,7 @@ import scalariform.formatter.preferences._
 
 name := "adstax-sdk-scala"
 organization := "eu.shiftforward"
-version := "0.4.0"
```
Comment (ruippeixotog) [inline build.sbt]: You need to change the README, too.

File: build.sbt [hunk 4]
```diff
@@ -3,7 +3,7 @@ import scalariform.formatter.preferences._
...
+version := "0.4.1"
```
Comment (ruippeixotog) [inline build.sbt]: You need to change the README, too.

File: src/main/scala/eu/shiftforward/adstax/ups/api/UserProfileStorageResponse.scala (hunk 1 above)
Comment (jcazevedo) [inline src/main/scala/eu/shiftforward/adstax/ups/api/UserProfileStorageResponse.scala]: Fixed in 0c5ef39.

File: src/main/scala/eu/shiftforward/adstax/ups/api/UserProfileStorageResponse.scala (hunk 2 above)
Comment (jcazevedo) [inline src/main/scala/eu/shiftforward/adstax/ups/api/UserProfileStorageResponse.scala]: Fixed in 0c5ef39.

File: build.sbt (hunk 4 above)
Comment (jcazevedo) [inline build.sbt]: Fixed in 07e166e.

**Comment by jcazevedo** [comment]
Just published `v0.4.1` of `adstax-sdk-scala`.

**Pull request merged**